pkg/controller/common/helpers_test.go

```diff
@@ -99,7 +114,12 @@ func TestParseAndConvert(t *testing.T) {
 
 	// Make a Ign2 comp config
 	testIgn2Config := ign2types.Config{}
-	tempUser2 := ign2types.PasswdUser{Name: "core", SSHAuthorizedKeys: []ign2types.SSHAuthorizedKey{"5678", "abc"}}
+	tempUser2SSHKeys := []ign2types.SSHAuthorizedKey{
```

Comment:
Minor: I think just having the above testing in `TestConvertIgnition2to3` should be enough. Unless you feel that this covers more use cases

Reply:
Fine with me!